dm/pkg/etcdutil/etcdutil_test.go

```diff
@@ -33,54 +35,56 @@ import (
 	"github.com/pingcap/tiflow/dm/pkg/terror"
 )
 
-var _ = Suite(&testEtcdUtilSuite{})
+var etcdTestSuite = SerialSuites(&testEtcdUtilSuite{})
```

Comment:
just a question, use `SerialSuites` instead of `ParallelSuites` (default) is for avoid some conflicts? shall we also need define `SetUpTest` or `TearDownTest` function for help?

Reply:
actully SerialSuites doesn't help to reduce datarace, it's a internal data-race bettew grpc server and etcd server 😂
but this comment help me finding out the workaround to solve this problem, add in  https://github.com/pingcap/tiflow/pull/4175/commits/6ea37fa20e76f2b79018cc4a10b6b067749bc6c9